### PR TITLE
fix/tooltip - Arrow

### PR DESCRIPTION
**Pull request by mellmiranda**
## Infos

[Task](https://juntossomosmais.monday.com/boards/448855787/pulses/687683504)

#### What is being delivered?

Changed CSS - Arrow width
Component Tooltip

#### What impacts?

Improved visualization for the user using the Tooltip component for any warnings.

#### Reversal plan

Commit revert

#### Evidences

tooltip_v2

## DoD checklist

- [ ] Related code finished
- [ ] Documentation updated (if exists)
- [ ] Unit tests written and passing
- [ ] Any configuration or build changes documented
- [ ] Project builds without errors
- [ ] Lint errors fixed

### CODE REVIEW

**Review by elizeumatheus — DISMISSED** [review]

File: packages/styles/components/Tooltip.module.scss [hunk 1]
```diff
@@ -58,15 +58,27 @@ $arrow-size: 5px;
...
+        &::before {
+          border-color: var(--gray-2) transparent transparent transparent;
+          border-style: solid;
+          border-width: ($arrow-size + 2px);
+          content: '';
+          left: 42%;
+          position: absolute;
+          top: 127%;
+          transform: translateX(-50%);
+          transform: translateY(-50%);
```
Comment (felipefialho) [inline packages/styles/components/Tooltip.module.scss:80]: `transform: translateY(-50%);` will overwrite `transform: translateX(-50%);`, maybe we should use `transform: translate(-50%, -50%)`

**Review by felipefialho — CHANGES_REQUESTED** [review]

File: packages/styles/components/Tooltip.module.scss (hunk 1 above)
Comment (mellmiranda) [reply packages/styles/components/Tooltip.module.scss]: Done :D

File: packages/styles/components/Tooltip.module.scss [hunk 2]
```diff
@@ -76,23 +78,24 @@ $arrow-size: 10px;
           left: 42%;
           position: absolute;
           top: 127%;
-          transform: translateX(-50%);
-          transform: translateY(-50%);
+          transform: translateX(-$coordinate-space);
+          transform: translateY(-$coordinate-space);
```
Comment (felipefialho) [inline packages/styles/components/Tooltip.module.scss:82]: We should to use `transform(-$coordinate-space, -$coordinate-space)` or `transform: translateY(-$coordinate-space);` will be overwrite `translateX(-$coordinate-space);`

File: packages/styles/components/Tooltip.module.scss [hunk 3]
```diff
@@ -76,23 +78,24 @@ $arrow-size: 10px;
           left: 42%;
           position: absolute;
           top: 127%;
```
Comment (felipefialho) [inline packages/styles/components/Tooltip.module.scss:80]: To avoid side effects is a good idea rewrite it avoiding to using magic numbers 😊

**Review by felipefialho — CHANGES_REQUESTED** [review]

File: packages/styles/components/Tooltip.module.scss (hunk 2 above)
Comment (mellmiranda) [reply packages/styles/components/Tooltip.module.scss:82]: Done, removed code:  transform: translateX(-$coordinate-space);

File: packages/styles/components/Tooltip.module.scss (hunk 3 above)
Comment (mellmiranda) [reply packages/styles/components/Tooltip.module.scss]: Done 🥰

**Review by hick97 — APPROVED** [review]

**Review by felipefialho — APPROVED** [review]

**Pull request merged**